README.md

```diff
@@ -1,2 +1,56 @@
 # Hello-world
 This is my first Microverse study repo.
+
+
+## Built With
+
+- Major languages - Html and CSS
+
+## Live Demo
+
+[Live Demo Link](https://github.com/white3d/Hello-world)
```

Comment:
You live demo link seem to be pointing directly to your repository. Kindly make sure it is a link to your hosted site or remove it if not hosted yet.